Runar/Runar/Controllers/Misc/TabBarController/MainTabBarController.swift

```diff
@@ -29,9 +37,9 @@ class MainTabBarController: UITabBarController {
     private func configureTabBar() {
         UITabBar.appearance().tintColor = Assets.TabBar.pushColor.color
```

Comment:
Let's set tintColor in tabBar property instead of static method

Reply:
If you don't mind, let's try to write it like this:
tabBar.tintColor = .yellowSecondaryColor